cg/cli/workflow/balsamic/base.py

```diff
@@ -275,12 +275,19 @@ def config_case(
 @click.option(
     "-r", "--run-analysis", "run_analysis", is_flag=True, default=False, help="start " "analysis",
 )
+@click.option(
+    "-a",
+    "--analysis-type",
+    "analysis_type",
+    type=click.Choice(["qc", "paired", "single"]),
+    required=False,
+)
```

Comment:
Could we add this as @ANALYSIS_TYPE_OPTION above? Something like this:
```suggestion
ANALYSIS_TYPE_OPTION = @click.option(
    "-a",
    "--analysis-type",
    "analysis_type",
    type=click.Choice(["qc", "paired", "single"]),
    required=False,
)
```

And add `@ANALYSIS_TYPE_OPTION` to `def balsamic(...)`and `def run(...)`.

Reply:
See @PRIORITY_OPTION and @EMAIL_OPTION for inspiration.